### DISCUSSION TITLE
Installing unlocked version vs namespaced version

### DISCUSSION THREAD

**Discussion by baobao917** [jamessimone/apex-rollup#486] [Q&A] [answered]
Looking to use Apex Rollups for the 1st time and had a hopefully simple question.  I see there are two versions:

1. Unlocked version
2. Namespace version

If installing the unlocked version, does this mean that if a new version/release of Apex Rollups becomes available, that we can't upgrade to it without uninstalling the original?  

Similarly if installing the namespace version, does that mean we will be able to upgrade the app without uninstalling the original?

Any other considerations b/w the two that needs to be considered?

Thanks!  Looking fwd to seeing if this is a better solution than DLRS!

**Comment by jamessimone** [comment, marked as answer]
Hey there, both namespaced and the unlocked (no namespace) versions support package upgrades without uninstalling first. I would only recommend using the namespaced version if you're working in an org that has a namespace; otherwise, it'll be a lot easier to simply use the base unlocked package.
